manual.asc

```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
 === Other Deck Actions
 Long tapping on a deck will show a list of other actions available to perform on that deck:
 
-Rename deck :: Use this option to rename a deck
+Add :: Use this option to create a new empty note to the seleted deck
+
+Browse cards :: Use this option to see the cards of the selected deck with the <<browser, card browser>>
+
+Rename deck :: Use this option to rename the selected deck
```

Comment:
```suggestion
Rename deck :: Renames the selected deck
```

Reply:
You may want to link to a section on deck naming here. This is how people add subdecks

Reply:
How about this change:
```
Rename deck :: Renames the selected deck

NOTE: You can create a subdeck in the selected deck, 
and also can make the selected deck a subdeck of another deck, 
by using the following naming convention: "PARENT::CHILD" (i.e. "French::Idiom").
For more information, please see 
https://docs.ankiweb.net/getting-started.html#decks[the section in Anki manual].
```

Reply:
That looks great, thanks @snowtimeglass